js/api-augment/definitions/frequencyTxPayment.ts

```diff
@@ -35,6 +35,7 @@ export default {
             type: 'FeeDetails',
           },
         },
+        version: 1,
```

Comment:
Not sure why this wasn't here, but it fixes the error and aligns it with the other definitions